### PR TITLE
Translate Souffle programs to Logicblox programs. 

### PR DESCRIPTION
**Pull request by b-scholz**
This converter was written by P. Subotic. I adapted and added it to the latest version of Souffle.

### CODE REVIEW

**Comment by mmcgr** [comment]
Updating /src/.gitignore would also be good.

**Comment by b-scholz** [comment]
Fixed it. Note that there is clang program called iwyu. I tried to run it on Souffle code / it did not work very well. However, if we get it to work, we can produce the smallest / correct include set for our program.

File: src/.clang-format [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
```
Comment (mmcgr) [inline src/.clang-format]: Did you mean to include this file? We already have a .clang-format in the project root. This one looks more like standard google formatting than souffle.

Comment (b-scholz) [reply src/.clang-format]: Initially - yes. I dumped clang-format-4.0 default settings in there. 
The reason is that not all systems have clang-format-4.0. With the format file we are independent of 4.0 / any version can be used. However, I failed to make it work.

**Comment by mmcgr** [comment]
Yo, Jenko me old mate, test this please.

**Comment by souffle-lang** [comment]
[0% (0.0%) vs master 0%](http://plang1.it.usyd.edu.au/jenkins/job/PR-Ubuntu-18-04/9/)

**Pull request merged**